cassandra/connection.py

```diff
@@ -443,33 +443,6 @@ class ProtocolError(Exception):
 class CrcMismatchException(ConnectionException):
     pass
 
-
-class ContinuousPagingState(object):
```

Comment:
I hope we're not removing a nice feature we should be copying rather than removing :-)
Datastax is pretty proud of this feature - https://www.datastax.com/blog/dse-advanced-performance-apache-cassandratm-goes-turbo

Reply:
Yeah... we'll get to it if we need it.